core/src/main/resources/reference.conf

```diff
@@ -1400,13 +1400,18 @@ datastax-java-driver {
     # By default, this thread is named after the session name and "-timer-0", for example
     # "s0-timer-0".
     timer {
-      # The timer tick duration
+      # The timer tick duration.
       # This is the how frequent the timer should wake-up to check for timed out tasks.
       # Lower resolution (i.e. longer durations) will leave more CPU cycles for running I/O
       # operations at the cost of precision of exactly when a request timeout will expire or a
       # speculative execution will run. Higher resolution (i.e. shorter durations) will result in
       # more precise request timeouts and speculative execution scheduling, but at the cost of CPU
       # cycles taken from I/O operations, which could lead to lower overall I/O throughput.
+      # Note for Windows users: the tick duration cannot be less than 10 milliseconds. If that is
+      # the case, the driver will use 10 milliseconds as the tick duration instead. Also, the tick
+      # duration must be a multiple of 10; if that is not the case, the driver will automatically
```

Comment:
where is that round down logic?

Reply:
https://github.com/netty/netty/blob/netty-4.1.28.Final/common/src/main/java/io/netty/util/HashedWheelTimer.java#L557-L564